shared/hybrid/AppDelegate.m

```diff
@@ -29,8 +29,7 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication*)application didFinishLaunchingWithOptions:(NSDictionary*)launchOptions
 {
-    self.viewController = [[MainViewController alloc] init];
-    return YES;
+    return [super application:application didFinishLaunchingWithOptions:launchOptions];
```

Comment:
This is what the AppDelegate.m generated by cordova actually looks like.
The super class (CDVAppDelegate.m) creates a window. You end up with our window and theirs, and theirs is the one visible.